README.md

```diff
@@ -193,11 +193,11 @@ Then copy the resulting baseline profile from the emulator to [`app/src/main/bas
 Run the following command to get and analyse compose compiler metrics:
 
 ```bash
-./gradlew assembleRelease -PenableComposeCompilerMetrics=true -PenableComposeCompilerReports=true
+./gradlew assembleRelease -PenableComposeCompilerReportsAndMetrics=true --rerun-tasks
 ```
 
-The reports files will be added to [build/compose-reports](build/compose-reports). The metrics files will also be 
-added to [build/compose-metrics](build/compose-metrics).
+The reports files will be added to ***build/module-name/compose-reports***. The metrics files will also be 
```

Comment:
I'm not sure what these triple stars mean. Usually, globbing for any directory is `**`.
Also, could you use code blocks for these file path to not make it appear like regular text?

Reply:
Oh I see, sorry for the confusion.
I still think using code block here would be better, but that is just nitpicking at this point. Feel free to ignore it

```markdown
added to `build/module-name/compose-metrics`
```

Reply:
Good suggestion. I'll accept and make it as code block.